.github/workflows/blog-links.yaml

```diff
@@ -0,0 +1,35 @@
+name: Blog Links
+
+on:
+  schedule:
+    - cron: "20 7 * * *"
+
+jobs:
+  check-links:
+    if: github.repository == 'canonical-web-and-design/ubuntu.com'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+
+      - name: Install linkchecker
+        run: sudo pip install LinkChecker
+
+      - name: Run linkchecker
+        run: |
+          linkchecker --recursion-level=2 \
+          	--check-extern --no-warnings https://ubuntu.com/blog \
+          	--no-follow-url '!https:\/\/ubuntu.com\/blog' \
+          	--ignore-url https://res.cloudinary.com \
+          	--ignore-url q_auto \
+          	--ignore-url fl_sanitize \
+          	--ignore-url c_fill \
+          	--ignore-url e_sharpen \
+          	--ignore-url w_[0-9]* \
+          	--ignore-url h_[0-9]* \
+          	--ignore-url https://ubuntu.com/security
```

Comment:
Shouldn't we know about these failures?

Reply:
@anthonydillon you picked up my PR before I added a proper description to my PR. I added some explanation now of the rules I am applying.

`/security` has it's own git action, so just in case I am preventing blog to follow those links. I guess blog posts could have security links, so perhaps excluding them is the wrong call.

Cloudinary rules are explained in the PR description. LinkChecker struggles parsing them correctly.

Reply:
The blog could still have typed links to security which would not be picked up by a security action.

Reply:
Removed the `--ignore-url https://ubuntu.com/security` from blog action
Removed the `--ignore-url https://ubuntu.com/blog` from security action